include/audio/aseq.h

```diff
@@ -0,0 +1,2344 @@
+/**
+ *  @file aseq.h
+ *  Zelda64 Music Macro Language (MML) Assembler Definitions
+ *
```

Comment:
is it correct to call this stuff "zelda64 mml" this stuff is a shared library isnt it?

Reply:
even for other games that use MML they are slightly different right? that was the impression I was under at least. like zelda MML is different than mario MML

Reply:
Yeah other games do use a mostly-similar format but there's small differences that this file doesn't support assembling for, so it's qualified with "Zelda64"